### ISSUE TITLE
Consider alternative file for connectivity

### ISSUE THREAD

**Issue by fnattino** [GO-Eratosthenes/dhdt#55]
It would be nice to drop the TXT format for a more efficient format  - @dicaearchus might have good idea for this

**Comment by fnattino** [comment]
Potential format: `.npy` https://towardsdatascience.com/what-is-npy-files-and-why-you-should-use-them-603373c78883 

Very suitable for array data, but does this behaves with headers and other attributes/metadata?

**Comment by dicaearchus** [comment]
yes, metadata is also given and the columns are of different datatypes.

another potential format: .parquet https://arrow.apache.org/docs/python/parquet.html
